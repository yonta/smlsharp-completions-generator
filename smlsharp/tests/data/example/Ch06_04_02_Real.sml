10.0;
1E2;
0.001;
~1E~2;
10;

10.0 + 1E1;
0.0 - 10.0;
10.0 / 2.0;
10.0 * 2.0;
