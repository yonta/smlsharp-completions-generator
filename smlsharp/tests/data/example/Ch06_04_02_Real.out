val it = 10.0 : real

val it = 100.0 : real

val it = 0.001 : real

val it = ~0.01 : real

val it = 10 : int

val it = 20.0 : real

val it = ~10.0 : real

val it = 5.0 : real

val it = 20.0 : real
